src/resources/Pipelines/ResultRankings/ResultRankingsInterfaces.ts

```diff
@@ -8,11 +10,27 @@ import {
 } from '../../Enums';
 
 export interface ResultRanking {
+    /**
+     * The unique identifier of the result ranking rule.
+     */
     id: string;
+    /**
+     * The attribute of the result ranking rule.
```

Comment:
What exactly is a result ranking rule attribute?

Reply:
It contains all the information related to a result ranking rule. I don't know really how to word that. 

It contains the name, id, condition, definition, etc.

Reply:
It would return a JSON of the rule config if I understand correctly? If that's the case, I would go with:

```suggestion
     * The configuration of the result ranking rule.
```